rotkehlchen/chain/evm/decoding/gearbox/decoder.py

```diff
@@ -66,20 +67,30 @@ def pools(self) -> dict[ChecksumEvmAddress, GearboxPoolData]:
     def _cache_mapping_methods(self) -> tuple[Callable[[DecoderContext], DecodingOutput]]:
         return (self._decode_pool_events,)
 
-    def _decode_common(self, context: DecoderContext, user_address_bytes: bytes) -> tuple[ChecksumEvmAddress, GearboxPoolData, FVal, FVal] | None:  # noqa: E501
-        """This function decodes the common data for the deposit and withdraw events."""
-        if not self.base.is_tracked(user_address := hex_or_bytes_to_address(user_address_bytes)):
-            return None
+    def is_weth_pool(self, token: EvmToken) -> bool:
+        return token.underlying_tokens[0].get_identifier(self.evm_inquirer.chain_id) in {
```

Comment:
this will raise indexError if the underlying tokens are misconfigured. Can you handle it?

Reply:
Handling it where it's called.